api/db-cdr/generate-cdr/bq-schemas/db_domain.json

```diff
@@ -15,12 +15,20 @@
         "type": "STRING"
       },
       {
-        "name": "domain_parent",
+        "name": "db_type",
         "type": "STRING"
       },
       {
       "name": "domain_route",
       "type": "STRING"
+      },
+      {
+        "name": "concept_id",
+        "type": "INTEGER"
+      },
+      {
+        "name": "count_value",
+        "type": "INTEGER"
```

Comment:
I couldn't comment on a big diff like criteria.csv. So this is regarding criteria.csv,, check that it has same num of rows and data looks same as in all-of-us-workbench-test cloud sql instance for cdr20180223 database.

Reply:
I just checked and the rows in the old and new files look right.